contrib/mesos/pkg/staticpods/gzip.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package staticpods
+
+import (
+	"bytes"
+	"compress/gzip"
+	"fmt"
+	"io/ioutil"
+
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/v1"
+)
+
+func GZip(pods <-chan *api.Pod) ([]byte, error) {
+	return gzipList(List(pods))
+}
+
+func gzipList(list *api.PodList) ([]byte, error) {
```

Comment:
ugh. i would hate to type gZip. i had settled on gzip and GZip because they
felt better. I'd rather change GZip to Gzip if we want absolute consistency

On Fri, Sep 25, 2015 at 2:01 AM, Dr. Stefan Schimanski <
notifications@github.com> wrote:

> In contrib/mesos/pkg/staticpods/gzip.go
> <https://github.com/kubernetes/kubernetes/pull/13943#discussion_r40401562>
> :
>
> > +
> > +import (
> > +	"bytes"
> > +	"compress/gzip"
> > +	"fmt"
> > +	"io/ioutil"
> > +
> > +	"k8s.io/kubernetes/pkg/api"
> > +	"k8s.io/kubernetes/pkg/api/v1"
> > +)
> > +
> > +func GZip(pods <-chan *api.Pod) ([]byte, error) {
> > +	return gzipList(List(pods))
> > +}
> > +
> > +func gzipList(list *api.PodList) ([]byte, error) {
>
> nit: if it's GZip, it should also be gZip....
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13943/files#r40401562>.
>